tests/syft/lib/torch/parameter_test.py

```diff
@@ -29,6 +29,31 @@ def test_parameter_vm_remote_operation() -> None:
     # assert len(alice.store._objects) == 0
 
 
+def test_get_copy() -> None:
+
+    alice = sy.VirtualMachine(name="alice")
+    alice_client = alice.get_client()
+
+    x = th.nn.Parameter(th.randn(3, 3))
+
+    xp = x.send(alice_client)
+
+    y = xp + xp
+
+    assert len(alice.store._objects) == 2
+
+    y.get_copy()
+
+    # no deletion of the object
+    assert len(alice.store._objects) == 2
+
+    # TODO: put thought into garbage collection and then
```

Comment:
Q: What should this do?

If you delete ```xp``` you should have one more element into the ```store``` (which is ```y```).
If you also delete ```y``` you should have 0.

Right? :smile:

Reply:
I copied the test above, it seems like the commented out part for delete doesn't work as expected, but it was already like that, perhaps we need to fix that up?

Reply:
The assert from line 54 should be ```assert len(alice.store_objects) == 1``` since we still have y

Reply:
Good spot, these are re-enabled and fixed now.